shared/Microsoft.Extensions.Process.Sources/ProcessHelper.cs

```diff
@@ -77,22 +72,22 @@ private static void GetAllChildIdsUnix(int parentId, ISet<int> children, TimeSpa
 
         private static void KillProcessUnix(int processId, TimeSpan timeout)
         {
-            string stdout;
             RunProcessAndWaitForExit(
                 "kill",
                 $"-TERM {processId}",
                 timeout,
-                out stdout);
+                out var stdout);
         }
 
-        private static int RunProcessAndWaitForExit(string fileName, string arguments, TimeSpan timeout, out string stdout)
+        private static void RunProcessAndWaitForExit(string fileName, string arguments, TimeSpan timeout, out string stdout)
         {
             var startInfo = new ProcessStartInfo
             {
                 FileName = fileName,
                 Arguments = arguments,
                 RedirectStandardOutput = true,
-                UseShellExecute = false
+                RedirectStandardError = true,
```

Comment:
We don't use stderr or out anywhere, just return a bool if the process didn't exit.

Reply:
stdout is used by pgrep to find the list of child processes

I'm capturing stderr (to suppress) because this was always producing random console output in dotnet-watch such as `kill 1234 failed: no such process`